app/forms/idv/phone_form.rb

```diff
@@ -131,7 +131,9 @@ def unsupported_delivery_methods(capabilities)
     end
 
     def user_phone?(phone)
-      MfaContext.new(user).phone_configurations.any? { |config| config.phone == phone }
+      MfaContext.new(user).phone_configurations.any? do |config|
+        PhoneFormatter.format(config.phone) == phone
```

Comment:
Are you guaranteed that `PhoneFormatter.format` has already been applied to `phone` or do you want to do that before starting the loop?

Reply:
It _should_ be in most cases from being called here: https://github.com/18F/identity-idp/pull/8936/files#diff-8670d5eff40bced29f912966380df25593659541b69b27d6f30d0437eb84f0fdL39